src/Ocelot/Configuration/Builder/RouteBuilder.cs

```diff
@@ -10,7 +10,8 @@ public class RouteBuilder
         private string _upstreamHost;
         private List<DownstreamRoute> _downstreamRoutes;
         private List<AggregateRouteConfig> _downstreamRoutesConfig;
-        private string _aggregator;
+        private string _aggregator;
+        private Dictionary<string, UpstreamHeaderTemplate> _upstreamHeaders;

```

Comment:
```suggestion
        private IDictionary<string, UpstreamHeaderTemplate> _upstreamHeaders;

```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1312/commits/c727b934719503710b60671f22b0db25c05207e4 ✔️